src/engine/input/key_mapping/KeyNames.cs

```diff
@@ -121,6 +121,35 @@ public static string Translate(uint keyCode)
             KeyList.Ssharp => "ß",
             KeyList.Division => "÷",
             KeyList.Ydiaeresis => "ÿ",
+
+            // Key names that would conflict with simple words in translations
+            KeyList.Forward => TranslationServer.Translate("KEY_FORWARD"),
+            KeyList.Tab => TranslationServer.Translate("KEY_TAB"),
+            KeyList.Enter => TranslationServer.Translate("KEY_ENTER"),
+            KeyList.Insert => TranslationServer.Translate("KEY_INSERT"),
+            KeyList.Delete => TranslationServer.Translate("KEY_DELETE"),
+            KeyList.Pause => TranslationServer.Translate("KEY_PAUSE"),
+            KeyList.Clear => TranslationServer.Translate("KEY_CLEAR"),
+            KeyList.Home => TranslationServer.Translate("KEY_HOME"),
+            KeyList.End => TranslationServer.Translate("KEY_END"),
+            KeyList.Left => TranslationServer.Translate("KEY_LEFT"),
+            KeyList.Up => TranslationServer.Translate("KEY_UP"),
+            KeyList.Right => TranslationServer.Translate("KEY_RIGHT"),
+            KeyList.Down => TranslationServer.Translate("KEY_DOWN"),
+            KeyList.Menu => TranslationServer.Translate("KEY_MENU"),
+            KeyList.Help => TranslationServer.Translate("KEY_HELP"),
+            KeyList.Back => TranslationServer.Translate("KEY_BACK"),
+            KeyList.Stop => TranslationServer.Translate("KEY_STOP"),
+            KeyList.Refresh => TranslationServer.Translate("KEY_REFRESH"),
+            KeyList.Search => TranslationServer.Translate("KEY_SEARCH"),
+            KeyList.Standby => TranslationServer.Translate("KEY_STANDBY"),
+            KeyList.Openurl => TranslationServer.Translate("KEY_OPENURL"),
+
```

Comment:
huh, why is this empty line here?

Reply:
Forgot to delete it when I was merging the lists I edited together...

Reply:
Removed.